config.go

```diff
@@ -21,13 +21,11 @@
 package config
 
 import (
-	"fmt"
+	"io/ioutil"
 	"os"
-	"path"
-	"path/filepath"
-	"sync"
+	"strings"
 
-	flag "github.com/ogier/pflag"
```

Comment:
Thanks! We don't use any particular features of that library, all I needed was POSIX compatibility.